Android/GooglePlaces/source/GooglePlaces/GooglePlaces.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="MSBuild.Sdk.Extras/2.0.54">
   <PropertyGroup>
-    <TargetFrameworks>MonoAndroid90</TargetFrameworks>
+    <TargetFrameworks>MonoAndroid11.0</TargetFrameworks>
```

Comment:
Please leave here 9.0.

11.0 will force all libraries and/or apps that depend/user this library to bump targetFramework and could make a lot of users unhappy.

Reply:
Thanks for feedback moljac, I also bumped the ANDROID_SDK_BUILD_TOOLS version from 28.0.3 to 30.0.2 in build.cake file, should I revert that too?

Reply:
this should be OK. (I think, not sure)

Reply:
@moljac I've reverted to monoandroid90 but I still had a query about whether you wanted me to revert the SDK build tools version back to 28.0.3?

Reply:
no 30.0.2 should be OK. We'll see soon. OK?

Reply:
Yep, I'm good.  What happens now?

Reply:
@moljac are you waiting for me to do something?  This is my first time so not too sure of the process.

Reply:
No. Working on huge AndroidX update. Hope to be back soon